weekly-updates/weekly-update.2015-05-15.md

```diff
@@ -0,0 +1,26 @@
+# io.js News
+
+We are happy to speak this news, **"io.js decides to join the Node Foundation"**. And our organization name will be renamed `nodejs`.
+
+As a first step, we will move from `iojs` organization to `nodejs` organization and will converge `joyent/node` gradually. We will continue to release `io.js` until the convergence have done.
+
+If you would like to see the convergence plan in detail, please check it at [jashkell repo](https://github.com/jasnell/dev-policy/blob/master/convergence.md). When you would like to ask questions, feel free to ask [@Fishrock123](mailto:fishrock123@rocketmail.com). The discussion log is [here](https://github.com/nodejs/io.js/issues/1664). 
```

Comment:
this is being renamed to `nodejs/node` as we speak. might want to hold off until it is.